lib/rules/no-unused-vars.js

```diff
@@ -99,7 +99,6 @@ module.exports = {
          * Determines if a given variable is being exported from a module.
          * @param {Variable} variable - EScope variable object.
          * @returns {boolean} True if the variable is exported, false if not.
-         * @private
```

Comment:
I'd prefer to have `@private`, as it will prevent JSDoc from generating docs for them if we ever get that working again. :)